docs/pages/blog/mui-core-v5.md

```diff
@@ -492,11 +496,69 @@ The following components are now available in the lab:
 
 ## v4 migration
 
-- The high-level changes required
-- Installation
-- ⚓️ We have introduced a new release line: v4.x.x-deprecations.x. This release line is kept in sync with the latest version of v4 and includes actionable deprecations to ease the migration to v5.
-- The codemod, covered a bit in https://material-ui.com/blog/2021-q2-update/
-- The migration guide
+We put a lot of work in order to make the migration from v4 to v5 as easier as possible. If you are starting your upgrade, these are the three things you should look into:
+
+- ⚓ We have introduced a new release line: v4.x.x-deprecations.x. This release line is kept in sync with the latest version of v4 and includes actionable deprecations to ease the migration to v5.
```

Comment:
I thought we discontinued `v4.x.x-deprecations.x`? Maybe say that developers can upgrade to the latest version of v4, it includes deprecations that developers can solve, moving them closer to v5?